README.md

```diff
@@ -17,50 +17,36 @@ Example video of pairing and handling.
 
 ## Configuration
 
-```
+```cpp
 //TCP for handling server port
 #define TCP_SERVER_PORT 5115
-//Maximum pairings available. Apple recommends 16. But be aware, one pairings needs (36+32)B of your EEPROM
+//Maximum pairings available. Apple recommends 16. But be aware, one pairing needs (36+32)B of your EEPROM
 #define MAX_PAIRINGS 16
 //if you are using EEPROM, you can set offset for Homekit to store pairings
 #define EEPROM_STORAGE_ADDRESS_OFFSET 512
-
```

Comment:
I believe the `#define`s below aren't used anymore, but are randomly-generated in `resetAll`, or hard-coded in the  accessory class…